### PR TITLE
Team2 group skills

### PR DESCRIPTION
**Pull request by hinckleyc**
We added information to:

Deadlines
Participation
Conduct
Conflicts
Infractions

Please let me know if I need to change anything to improve?

### CODE REVIEW

**Review by enpuyou — CHANGES_REQUESTED** [review]
@hinckleyc Please reference to https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-markdown for the style of assessment. Also, there is an inconsistent use of bold texts. If your team believe that this will help with the readability, you may want to raise an issue for the style guide or contact other students on the markdown style guide team for the change

**Comment by koscinskic** [comment]
I have opened an issue on the Markdown team repository concerning the usage of bold text as well as resolved the requested changes concerning emoji style of assessment.

**Comment by hodevin** [comment]
There are still some inconsistent use of period punctuation but other than that, it looks great.

File: assessment.md [hunk 1]
```diff
@@ -81,15 +81,86 @@ Project Management is Covered by Team 2
...
+    time before the deadline to properly test their code/work and enough time to
+    fix any errors that may have arisen
+  * :sunglasses: = Team member **completes work on time** and encourages others
+    to also complete their work with enough time to test it and fix errors.
+
+A software engineer demonstrates the mastery of this professional skill at the
+E level by completing their assigned work for the project with enough time
+remaining to properly test and document it, and enough time to fix any errors.
+Additionally, they should encourage others to complete their work in
+a similar time frame such that the project flows as smoothly as possible.
```
Comment (corlettim) [inline assessment.md]: Should this still be here since you have now begun the use of emojis instead of the letters?

File: assessment.md [hunk 2]
```diff
@@ -81,15 +81,86 @@ Project Management is Covered by Team 2
...
+  * :sunglasses: = Team member **actively contributes** to the project by
+    **fulfilling their role / position** and **effectively communicates** with
+    their team. The team member **actively seeks to resolve issues** and
+    **guide** other team members in matters concerning their role / position.
+
+A software engineer demonstrates the mastery of this professional skill at the
+E level by understanding their role / position within the group and handling
+aspects of the project that concern this role / position. Furthermore, they
+will effectively communicate their intentions and abilities in order to
+maximize productivity.
```
Comment (corlettim) [inline assessment.md]: Same as above...this may not be needed

Comment (koscinskic) [reply assessment.md]: I changed the "E" to the appropriate emoji substitute on both branches, but I am still under the impression that an example is warranted regardless of the symbolic representation. The suggestion for an example blurb is in the original README.md

**Comment by noorbuchi** [comment]
Hi, I'm a member of the markdown team and I've been trying to review some of the pull requests and make sure that the branches adhere to the markdown style in the markdown repository . There is a small issue with your pull request in which you are using bold in your writing. Please let me know if you any questions about markdown usage. Thanks!

**Comment by koscinskic** [comment]
After reviewing the Markdown repository once more, I have found no guidelines governing the usage of modified text. If team members should not be using bold text under any circumstances, please specify this in the Markdown repository. Otherwise, this is an invalid issue that should not prohibit the merging of this PR.

**Review by corlettim — APPROVED** [review]
this looks good!

**Comment by noorbuchi** [comment]
@koscinskic the markdown repository has recently been updated to demonstrate the fact bold should not be used. Additionally, there was a recent announcement notifying everyone of this change.

**Comment by koscinskic** [comment]
All of the guidelines should be adhered to, thank you for the clarification.

File: assessment.md [hunk 3]
```diff
@@ -79,15 +79,110 @@
...
+    time before the deadline to properly test their code/work and enough time to
+    fix any errors that may have arisen
+  * :sunglasses: = Team member completes work on time and encourages others
+    to also complete their work with enough time to test it and fix errors.
+
+A software engineer demonstrates the mastery of this professional skill at the
+:sunglasses: level by completing their assigned work for the project with enough
+time remaining to properly test and document it, and enough time to fix any
+errors. Additionally, they should encourage others to complete their work in a
+similar time frame such that the project flows as smoothly as possible.
```
Comment (enpuyou) [inline assessment.md]: This section needs to be removed according to the latest assessment style guide

File: assessment.md [hunk 4]
```diff
@@ -79,15 +79,110 @@
...
+  * :sunglasses: = Team member actively contributes to the project by
+    fulfilling their role / position and effectively communicates with
+    their team. The team member actively seeks to resolve issues and
+    guide other team members in matters concerning their role / position.
+
+A software engineer demonstrates the mastery of this professional skill at the
+:sunglasses: level by understanding their role / position within the group and
+handling aspects of the project that concern this role / position. Furthermore,
+they will effectively communicate their intentions and abilities in order to
+maximize productivity.
```
Comment (enpuyou) [inline assessment.md]: Same here

**Review by enpuyou — CHANGES_REQUESTED** [review]
The period in each bullet point line needs to be removed

**Review by enpuyou — APPROVED** [review]

**Review by gkapfham — APPROVED** [review]
Thanks for your work on this PR, I approve you to merge it to the master branch.

**Review by koscinskic — APPROVED** [review]

**Pull request merged**